editor/editor_command_palette.cpp

```diff
@@ -107,14 +129,17 @@ void EditorCommandPalette::_update_command_search(const String &search_text) {
 		TreeItem *ti = search_options->create_item(root);
 		ti->set_text(0, TTR("No Matching Command"));
 		ti->set_metadata(0, "");
+		Color c = Color(0.5, 0.5, 0.5, 0.5);
+		ti->set_custom_color(0, c);
 		search_options->deselect_all();
 
 		get_ok_button()->set_disabled(true);
 	}
 }
 
 void EditorCommandPalette::_bind_methods() {
-	// to-do: bind methods.
+	ClassDB::bind_method(D_METHOD("add_command", "command_name", "key_name", "callable", "arguments", "shortcut_text"), &EditorCommandPalette::add_command, DEFVAL(varray()), DEFVAL("None"));
+	ClassDB::bind_method(D_METHOD("remove_command", "key_name"), &EditorCommandPalette::remove_command);
```

Comment:
I think `key_name` in `add_command()` should be `section_name`, although maybe I misunderstand it's purpose.

But in `remove_command` it should definitely be `command_name`.

Reply:
I think for `remove_command` it should be `key_name` because this function just removes the `Command` from the `commands: HashMap<String, Command>` and `command_name` is a member of `Command` Struct.

Reply:
Ah, I thought it's the same as `command_name` in `add_command()`. Nevermind then.